docs/update.md

```diff
@@ -26,7 +26,11 @@ This is a detailed documentation for the update functionality of packit.
     * Once your upstream release is out (and the spec file is really up to
       date), you can use packit to release it into Fedora.
 
-3. Pagure dist-git is configured in a way that it requires 2 API tokens in
+3. Create a new upstream release. The spec file needs to be included in the ref
```

Comment:
I would highlight it a bit because it is important. Something like:
`THIS IS IMPORTANT STEP` otherwise `packit` fails.

Reply:
all the steps are equally important here: if you won't do them, packit will fail